CONTRIBUTING.rst

```diff
@@ -10,7 +10,7 @@ take a moment to read our suggestions for happy maintainers and
 even happier users.
 
 The ``ev3dev-stretch`` branch
----------------------
+-----------------------------
```

Comment:
The looks like it is too long by 4 `-`s since the 4 single back ticks in the line above are not displayed in the final output.

Reply:
Actually do we need to make these match? When I view the current doc here:
https://github.com/ev3dev/ev3dev-lang-python/blob/ev3dev-stretch/CONTRIBUTING.rst

The `---`s just appear as a single line all the way across the page.

Reply:
The only reason I extended the dashes is because the reStructuredText linter extension in Visual Studio Code flagged it. The sphinx renderer likely doesn't care either way... but darn, THAT LINTER!